pkgs/top-level/all-packages.nix

```diff
@@ -7372,14 +7372,22 @@ with pkgs;
     inherit (darwin) apple_sdk;
   };
 
-  # For beta and nightly releases use the nixpkgs-mozilla overlay
-  rust = callPackage ../development/compilers/rust ({
-    inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
+  rustBootstrap = recurseIntoAttrs (makeRustPlatform (callPackage
```

Comment:
Oh sorry, rather than changing references to `rustBootstrap`, `makeRustPlatform` could use `buildPackages.callPackage` here. I'd have to stare at the details longer, but the gist of it is all the compilers should come from the previous stage (`buildPackages`).

Reply:
Here too? https://github.com/NixOS/nixpkgs/pull/50835/files#diff-036410e9211b4336186fc613f7200b12R13026

Reply:
What is the practical difference of doing so?